torch/csrc/autograd/engine.cpp

```diff
@@ -271,6 +280,10 @@ auto Engine::thread_main(GraphTask *graph_task) -> void {
   // Note [Reentrant backwards]
   while (!graph_task || graph_task->outstanding_tasks_ > 0) {
     FunctionTask task = queue->pop();
+    if (task.isShutdownTask_) {
+      C10_LOG_API_USAGE_ONCE("worker shutting down");
```

Comment:
Based on grepping existing sites, it looks like these want to be machine readable strings with no spaces. Maybe something like `torch.autograd.thread_shutdown`?